### PR TITLE
feat(curriculum): add animations to block 5.3 of the English Curriculum

### PR DESCRIPTION
**Pull request by DanielRosa74**
Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-tech-trends-and-updates/66360374d980d82d3cccddf7.md
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-tech-trends-and-updates/66360374d980d82d3cccddf7.md:1]: Tasks 6 and 7 uses two long sentences. I suggest we break those into smaller tasks. Here's what I'd do:
Create new tasks that would come first and be new Task 6 and 7 for the sentence "Low-code development is about making software with less code." 
Task 6 fill in the blanks. blanks: making, less. teach what "make something less" mean.
Task 7: Check understanding of the sentence. 

The current tasks 6 and 7 would become Task 8 and 9 and focus solely on this part:  "It's one of those trends that simplifies the development process.". Task 9 question would have to change to: "What does low-code development do according to Sarah?"

**Review by nieldakarla — COMMENTED** [review]
Hey Dani! great work with the animations. COuldn't find any problems with the audio. 
I do have a suggestion on some tasks as follows:

Tasks 6 and 7 uses two long sentences. I suggest we break those into smaller tasks. 

Here's what I'd do:
Create new tasks that would come first and become Task 6 and 7 for the sentence "Low-code development is about making software with less code."

Task 6 fill in the blanks. blanks: making, less. teach what "make something less" mean.
Task 7: Check understanding of the sentence.

The current tasks 6 and 7 would become Task 8 and 9 and focus solely on this part: "It's one of those trends that simplifies the development process.". 

Task 9 question would have to change to: "What does low-code development do according to Sarah?"

What do you think?

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-tech-trends-and-updates/66360374d980d82d3cccddf7.md
Comment (DanielRosa74) [reply curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-tech-trends-and-updates/66360374d980d82d3cccddf7.md:1]: On it. Thanks, @nieldakarla!

**Review by nieldakarla — APPROVED** [review]
This looks perfect! Thank you for the adjustments! =)

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**